cas: 54
injGnd: 1108364
reqPd: 16777215
puX: 8159221
diffOff: 1783
req: 159147
refr: 6
puY: 16777215
diffOn: 482443
diff: 30153
foll: 51
Pr: 3
